webapp/channels/src/components/threading/global_threads/thread_list/thread_list.tsx

```diff
@@ -251,6 +279,8 @@ const ThreadList = ({
                 )}
             />
             <div
+                id='threads-list'
+                role='tabpanel'
```

Comment:
In case this comes up later, I chose not to add an `aria-labelledby` here pointing back to the Followed/Unread button which is recommended by the APG for this. We don't switch between multiple tab panels, and it's not essential to add a label to this (it's a SHOULD and not a MUST in the WAI spec), so I think it's probably fine to diverge from the recommendations here